src/views/Partner.vue

```diff
@@ -125,7 +125,8 @@ export default {
       formData.append('email', this.email)
       formData.append('summary', this.summary)
       formData.append('site', this.site)
-      axios.post('https://guarded-ravine-42139.herokuapp.com/partner-form', formData)
+      alert('submitting')
+      axios.post('http://localhost:5000/partner-form', formData)
```

Comment:
Is this supposed to be in production? I'm not familiar with this code, but `localhost:5000` doesn't look right

Reply:
I completely agree that slipped through haha. I'm closing this PR anyway since Karuna was a bit finicky of a project partner.